README.md

```diff
@@ -26,6 +26,9 @@ Here is an example of how you can see the list of issues and the issue descripti
 To start using `hit` make sure that you have the following tools installed on your machine:
 
 * [ `git`][git] — `hit` is a wrapper around `git`
+* `diff-highlight` — for pretty output of the `hit diff` command
```

Comment:
Should somehow highlight that this is optional, but recommended.

Reply:
Good idea! I can add a bold prefix like `**Optional**`. What do you think?